### PR TITLE
Use aerie logo with background in readme

### PR DESCRIPTION
**Pull request by JoelCourtney**
## Description
The logo in the README is nearly invisible in Github's dark theme (for me at least; I'm colorblind). I added a version of the logo wordmark that includes a white background around the logo, and added that to the readme.

5px margin and 2px rounded corners.

@lklyne you're the design king so you have veto power.

## Verification
I think it looks nifty.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <br>
 <div align="center">
-  <img alt="Aerie" height="50" src="docs/sphinx_scylladb_theme/static/img/logos/aerie-wordmark-light.svg">
+  <img alt="Aerie" height="50" src="docs/sphinx_scylladb_theme/static/img/logos/aerie-wordmark-with-background.svg">
```
Comment (camargo) [inline README.md]: It seems to be rendering a little smaller than what is on develop. Not sure why. Can we make it the same size?

Comment (JoelCourtney) [reply README.md]: Ah that's because I widened the SVG but not the img tag.

Comment (JoelCourtney) [reply README.md]: Increased the img height to 60.

Comment (camargo) [reply README.md]: It still feels a little too small for the page. Maybe increase to like 80-100?

Comment (JoelCourtney) [reply README.md]: Increased the height to 85.

**Review by camargo — APPROVED** [review]
LGTM

**Pull request merged**